io.h

```diff
@@ -75,7 +75,12 @@ struct io_ring_ctx {
 		unsigned		sq_mask;
 		unsigned		sq_thread_idle;
 
-		struct io_ring_queue	*queue;
+		void    *queue;
```

Comment:
What happened here? Did io_ring_queue get deprecated on certain kernels? 
All changes look fine, a note regarding why this is required would be great.

Reply:
This is px-fuse defined struct. This has been reorg for the same reason to split into config init and uring use.